lib/documents.js

```diff
@@ -37,7 +37,11 @@ function Paragraph(children, properties) {
         styleId: properties.styleId || null,
         styleName: properties.styleName || null,
         numbering: properties.numbering || null,
-        alignment: properties.alignment || null
+        alignment: properties.alignment || null,
+        indentLeft: properties.indentLeft || null,
+        indentRight: properties.indentRight || null,
+        indentFirstLine: properties.indentFirstLine || null,
+        indentHanging: properties.indentHanging || null
```

Comment:
I think the code should be simplified to this:

```JS
function Paragraph(children, properties) {
    var paragraph= properties || {};
    paragraph.type = types.paragraph;
    paragraph.children = children;
    return paragraph;
}
```

I don't get why the properties are double filtered/white-listed. (The fact that the parser picked it up should be enough).